test/markdownlint-cli2-test-cases.js

```diff
@@ -629,15 +637,19 @@ const testCases =
       "invalid.markdownlint-cli2.jsonc",
       /(?:Unexpected end)|(?:Expected property name)/u
     ],
-    [ "invalid.markdownlint-cli2.cjs", /Unexpected end of input/u ],
-    [ "invalid.markdownlint-cli2.mjs", /Unexpected end of input/u ],
+    // eslint-disable-next-line max-len
+    [ "invalid.markdownlint-cli2.cjs", /Unable to require or import module '.\/invalid.markdownlint-cli2.cjs/u ],
+    // eslint-disable-next-line max-len
+    [ "invalid.markdownlint-cli2.mjs", /Unable to require or import module '.\/invalid.markdownlint-cli2.mjs/u ],
     [
       "invalid.markdownlint.json",
       /(?:Unexpected end)|(?:Expected property name)/u
     ],
     [ "invalid.markdownlint.yaml", /Map keys must be unique/u ],
-    [ "invalid.markdownlint.cjs", /Unexpected end of input/u ],
-    [ "invalid.markdownlint.mjs", /Unexpected end of input/u ]
+    // eslint-disable-next-line max-len
+    [ "invalid.markdownlint.cjs", /Unable to require or import module '.\/invalid.markdownlint.cjs/u ],
```

Comment:
Please escape the "." character in these RegExps.

Reply:
done